conf/squid/common/4node_1client_2rgw_3way_ms.yaml

```diff
@@ -0,0 +1,114 @@
+# Conf for 2+2 EC profile on 4 nodes for 3 way active-active rgw-multisite
```

Comment:
Common cluster configurations will be used, don't need to these global-confs

Reply:
Hi @vamahaja this pr is being used as reference, chnanges will be raised in separate pr, As cephci team also dependent on the same pr for conf and deployment details